### DISCUSSION TITLE
Copilot Markdown file

### DISCUSSION THREAD

**Discussion by barum** [community/community#7146] [Copilot Conversations]
in a Markdown (.md) file, Copilot shows the recommendations. However, there is an issue with <TAB> to complete.
The TAB key works as normal and does not complete/accept the recommendations. I have to hover over the recommendations and press accept.

**Comment by sophos-rickc** [comment]
@barum I'm glad you mentioned hover and [Accept]. I hadn't figured that out as an alternative to [Tab]... Thanks!

**Comment by jeffrafter** [comment]
To fill this out more - in my VSCode settings I have:

```
  "github.copilot.enable": {
    "*": true,
    "yaml": false,
    "plaintext": true,
    "markdown": true
  }
```

And then I have changed my tab keyboard shortcut in markdown to `Opt+Tab`:

![image](https://user-images.githubusercontent.com/4064/140822791-13eb5769-2f89-4239-ac55-3ff18a42f356.png)

**Reply by jeffrafter** [reply]
Some of the Tab key functionality was coming from the Markdown all-in-one extension.

**Comment by Sti2nd** [comment]
I had the same issue and disabling the extension [Markdown All In One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) fixed it. Probably the extension adds a TAB keyboard inside markdown files.

**Comment by jrummell** [comment]
There's a workaround at [Respect Inline Suggestion to improve compatibility with GitHub Copilot](https://github.com/yzhang-gh/vscode-markdown/issues/1011#issue-996151712)
